src/providers/wfs/qgswfssourceselect.h

```diff
@@ -103,7 +103,8 @@ class QgsWFSSourceSelect: public QgsAbstractDataSourceWidget, private Ui::QgsWFS
 
     void populateConnectionList();
 
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
+    void on_buttonBox_helpRequested() { QgsHelp::openHelp( QStringLiteral( "working_with_ogc/ogc_client_support.html" ) ); }
+
 
```

Comment:
This `on_buttonBox_helpRequested()` is never called.

When you click the Help button in the new connection dialog, the method that is called is `on_buttonBox_helpRequested()` in the `src/gui/qgsnewhttpconnection.h`

This change does not create any problem, but does nothing. For me it is ok to accept this since we will change this behaviour in a more extensive refactor of all help calls.